### PR TITLE
fix: add alt text to image and label to improve code’s accessibility

### LINKED ISSUE #51: [a11y]: labels and alt text are missing
### Description 
In the [HTML file for the lines & blocks lesson](https://github.com/codedex-io/css-101/blob/main/4-layout/19-lines-and-blocks/index.html), there are a couple of accessibility issues that I found:

1. The image is missing alt text. This makes it difficult for screen reader users to understand what the image is describing and how it fits the exercise 
2. The inputs are missing associated labels, which causes screen reader users to [not understand its purpose](https://www.w3.org/WAI/tutorials/forms/labels/)

### PR DESCRIPTION
**Pull request by CBID2**
### Description 
This PR adds alt text and labels in the HTML file for the lines and blocks lesson, making the code easier for users with disabilities to interact with.

### Issue
Closes #51

### CODE REVIEW

**Comment by intelagense** [comment]
This is good and I can also update the starter code but can you double check because it looks like submit button is missing and the formatting is off. 🙏

File: 4-layout/19-lines-and-blocks/index.html [hunk 1]
```diff
@@ -38,10 +38,12 @@ <h2>Services</h2>
       <h2>Contact</h2>
       <p>Interested in upskilling with Rendezvous Tutoring? Feel free to sign-up with your e-mail below to learn more!</p>
       <form>
-        <input type="email" placeholder="e-mail here" required />
-        <input type="submit" value="Sign Up" />
+      <label for="email">Email:</label>
+<input type="email" id="email" placeholder="e-mail here" required />
+        <button type="submit"> Submit</button>
```
Comment (CBID2) [inline 4-layout/19-lines-and-blocks/index.html:43]:
```suggestion
        <button type="submit"> Sign up</button>
```

**Comment by CBID2** [comment]
> This is good and I can also update the starter code but can you double check because it looks like submit button is missing and the formatting is off. 🙏

Fixed it @intelagense

File: 4-layout/19-lines-and-blocks/index.html [hunk 2]
```diff
@@ -38,10 +38,12 @@ <h2>Services</h2>
       <h2>Contact</h2>
       <p>Interested in upskilling with Rendezvous Tutoring? Feel free to sign-up with your e-mail below to learn more!</p>
       <form>
-        <input type="email" placeholder="e-mail here" required />
-        <input type="submit" value="Sign Up" />
+      <label for="email">Email:</label>
+<input type="email" id="email" placeholder="e-mail here" required />
+        <button type="submit"> Sign up</button>
```
Comment (intelagense) [inline 4-layout/19-lines-and-blocks/index.html:43]: Let’s keep this as an `<input>` for now to match what was taught in the HTML course and avoid confusion, even though `<button>` elements might be a better choice overall.

**Review by intelagense — CHANGES_REQUESTED** [review]

File: 4-layout/19-lines-and-blocks/index.html (hunk 2 above)
Comment (CBID2) [reply 4-layout/19-lines-and-blocks/index.html:43]: Ok.

**Comment by CBID2** [comment]
Made the change @intelagense

**Pull request merged**